cip-0008.md

```diff
@@ -0,0 +1,56 @@
+
+## Motivation ##
+
+Implementation of most CIPs requires a non-trivial amount of highly skilled design and development time. Bounties raised from the Counterparty community can be a useful tool to promote implementation of specific CIPs via raising the funds sometimes necessary to incentivize talented developers and compensate them for their time. This is especially true given the non-profit, volunteer nature of Counterparty development in general.
+
+## Rationale ##
+
+Prior to this process, implementation of CIPs via bounties was performed somewhat ad-hoc. This raised some confusion and exposed several ambiguities, such as potential mismatches in expectations between the individuals donating to the effort and the parties implementing the effort, as well as confusion regarding the degree of latitude to which the development party and/or project maintainers could exercise creative control of the implementation.
+
+The goal of this CIP is to remove these and other potential points of conflict and institute a process that maximizes the chance of success while remaining as lightweight as possible.
+
+## Specification ##
+
+### CIP Custodians ###
+
+The Counterparty Foundation Board may appoint one or more CIP Custodians, who shall be in charge of handling funds for currently open CIP bounty efforts. CIP custodians shall serve for an indeterminate amount of time and may be freely appointed and removed by the Counterparty Foundation members via a majority vote.
+
+In its fiduciary duty, the Board shall seek to only appoint individuals that have a both a proven track record of involvement in Counterparty, as well as demonstrate trustworthy character traits. Each CIP custodian may not serve as a Counterparty Foundation board member at the same time, and may not serve as both a CIP author and CIP custodian for any specific proposal.
+
+### Bounty provisioning and operation ###
+
+To begin, a CIP is created as normal, with the "Draft" status assigned. Once a CIP is in the draft stage, the CIP's author may choose to contact a CIP custodian, who will provision a donation address for that bounty under the sole control of the CIP custodian. A separate private key/address shall be used to handle the funds for each bounty independently. If multiple CIP custodians are serving, they may choose to jointly secure funds for a specific bounty via a multi-signature scheme, but such a scheme is not required. All funds gathered shall stay at the address to which they were sent while under the control of a CIP custodian.
+
+Before a CIP custodian may provision a donation address for a specific CIP, that CIP must be written to include a section entitled "Milestones", which shall include the following:
+
+* The fundraising goal, being a total amount (expressed in BTC, XCP or USD) that must be raised in order to consider fundraising for the CIP complete. (Contributions over and above this amount will be refunded to their senders by the CIP custodian.)  
```

Comment:
USD is tricky here because the funds may be raised, and then the price of BTC or XCP could change causing there to be too much or too little money.  Should we just remove the USD option?